addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -13,6 +13,9 @@ $gray-900: #212529 !default;
 $white: #FFF !default;
 $black: #000 !default;
 
+$light: o-color('o-cc2-bg');
+$dark: o-color('o-cc5-bg');
```

Comment:
> Could you sum up the rational behind this?

 So, as you know we currently have two "light" colours: 
- Bootstrap's `$light` has a widespread visual impact, but it's not customisable -> several elements are locked to a specific design that may mismatch the user's palette . 
- `cc2`, instead, it's customisable but it's not (really) used.

This resulting behaviour is inconsistent because, while the correlation `cc1= $primary` and `cc3 = $secondary` is fairly apparent,  changing `cc2` has no visual result on screen instead.

At the same time you have several BS components that are "locked" for no good reason.
The `$light` variable is exposed at the root of BS's palette system, like `$primary` etc... it's meant to be adapted accordingly to the rest.

Switching between the default palettes provides a much better visual result compared to master.
 
Ultimately this change will provide better integration with bootstrap and a more predictable edition.
The same logic applies for $dark evidently.

> still risky especially in old databases which may have both cc2-bg and cc5-bg being dark or light

There is a design reason why we have exactly 5 colours and it's not a coincidence that this number matches the BS ones.
In a "normal website" cc2 need to be somehow "light".

Having said that, the user choices and he's obviously free to do whatever he likes but, more important, testing this new system we have been able to create a "dark-mode" palette very easily; IMO it's an hint that the proposed system is a step further in terms of consistency and BS integration and the the user has at the more more flexibility to solve undesired behaviours
![image](https://github.com/odoo/odoo/assets/8558351/b020583c-0339-4aca-bdfb-b8078dad7fe8)

Reply:
@stefanorigano I agree, for the most part. If we started with this with some kind of indication that cc2 should be light and cc5 should be dark.

> cc2, instead, it's customisable but it's not (really) used.

Not really true: in old database maybe someone applied cc2 everywhere and switching to 17.0 would break its whole website. Same for a new user that would start using the website, customize cc2, use it then install eCommerce later where $light is way more used... and see it does not like it with no real solution other than redoing its website pages.

> Bootstrap's $light has a widespread visual impact, but it's not customisable

You could already control its tint and saturation but indeed not its lightness. IMO providing a way to scale the gray palette (saying gray-100 starts way darker) would be safer than doing this.

> In a "normal website" cc2 need to be somehow "light".

The problem here can be summed up to: "yes but people don't know that"

I am not against this change, I just want to be sure everybody knows the problems and are sure it is worth it taking them. cc @pierre-pde @rdeodoo 
In the end what we could also do is adding light and dark colorpickers in debug mode which would override light / dark coupling to cc2 and cc5. That way migrated users could use that if complaining :shrug: But it's a bit more code for maybe a problem that will not occur.

Reply:
Follow-up after the cc4-bg impacting pagination:

However if we do this, I would go for color-2 and color-5 and not cc2-bg and cc5-bg.

Also why 2 and 5 and not 4 and 5... would not it be more predictable to use the two last ones which are light and dark visually?

Reply:
> >In a "normal website" cc2 need to be somehow "light".
> 
> The problem here can be summed up to: "yes but people don't know that"

I wasn't aware of that myself. To me those are just preset of colors that I like so I can have a short cut to apply it where I want it, and that on top of that it will reflect any change I made to this preset later.

That's IMHO something only 2-3 people in Odoo will know, and 0% of our clients :grimacing: 

People will (and do) change those color randomly until they got a set of colors they like, that's pretty much about it, we can't expect them to do better (and I don't think we want them to).

If we want to really say "this color **should** be the one used for light color", then we should inform the user but that doesn't really makes sense to me as an end user.

When discussing this with QSM yesterday, I told him that if we did that, we could do stuff like preventing the selection of a dark color for this one (they are disabled) and warn the user, but it would be terrible UX wise on top of not being correct, as explained to me yesterday by QSM as your light color could be somehow dark~ish if you have a very dark/black website.

Hopefully I got what you were discussing right :)

Reply:
Just to clarify:
- I have ~10% knowledge at best of the color picker specs
- I don't bring any alternative solution
- It was just an opinion of this particular topic without taking anything else into consideration
- Here is the concern in video: https://watch.screencastify.com/v/a8h4GWUXhHRp25Idrhu7 Sorry a bit long but it should highlight what I meant

Reply:
> cc2, instead, it's customisable but it's not (really) used.
>> Not really true: in old database maybe someone applied cc2 everywhere and switching to 17.0 would break its whole website.

I'm not saying that "people never clicked that option", what I'm saying is that changing the value doesn't really have an impact on how the system is designed (or BS is compiled if you prefer). The only changes are within the snippets that uses that color combination, that's it.
No changes on buttons, no changes on Bootstrap or odoo components, for example.
In comparison to change `color-1` has much more deeper effect.

> Same for a new user that would start using the website, customize cc2, use it then install eCommerce later where $light is way more used... and see it does not like it with no real solution other than redoing its website pages.

It's the opposite actually 😉. With this PR you change `color-2`and you see your changes applied in forms, buttons and sidebars... and when you install eCommerce all its buttons are styled accordingly, like all the other apps 😉 (forum, appointments, partners, have been already adapted with $light... portal and events will come soon)

>In a "normal website" cc2 need to be somehow "light".
>>The problem here can be summed up to: "yes but people don't know that"

I was referring to general guidelines to create a standard website, but let's forget about that 😉
People don't have to know that, let's says that's a secret. They already start from default palettes that follows these rules but they deviate as much as they want, it's fine.
My point is that with this system they can deviate **much more**... check the previous screenshot.. can you achieve that in master?

Reply:
> can you achieve that in master?

Like I said, I agree this is sometihng we want :wink: We'll call you :)

Reply:
Conclusion of the talk:
- We'll link light and dark to color-2 and color-5 instead of cc2-bg and cc5-bg. That way people can still have 5 presets with 5 custom background without impacting other apps and this eases migration (if color-2 was changed to "red" to have a red preset, you can revert it back to the "light" you want and force that red preset _background_ to red itself). For the bg-o-color-2 class uses, this will need to be manually updated snippet by snippet though but this is not the main good way to change an area background anyway. This is also easier to understand that the 5 themes colors impacts components rather than presets backgrounds.
- Instead of the group color-1 color-3 color-2    +    color-4 color-5, we'll have color-1 color-2    + color-3 color-4 color-5 with the two groups being named "main" and "light and dark".